FirebaseRemoteConfig/Sources/RCNConfigDBManager.h

```diff
@@ -110,6 +112,15 @@ typedef void (^RCNDBLoadCompletion)(BOOL success,
 /// Insert or update the data in Personalization config.
 - (BOOL)insertOrUpdatePersonalizationConfig:(NSDictionary *)metadata fromSource:(RCNDBSource)source;
 
+/// Insert rollout metadata in rollout table.
+/// @param key        The key of rollout metadata belongs to, which are defined in
```

Comment:
Since the "key" here is really just active or fetched metadata (rather than a typical DB primary key) I wonder if we could improve the documentation to indicate that

Reply:
Changed the doc to be more specific.